.github/workflows/buf-update.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Run 'buf update'"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 10 * * 1'
+
+jobs:
+  update-buf-dependencies:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run 'buf update'
+        run: |
+          export PATH=$PATH:$(go env GOPATH)/bin
+          make install-tools proto-update
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v6.0.2
```

Comment:
we might want to use the latest version while we're at it https://github.com/peter-evans/create-pull-request/releases/tag/v6.1.0